amethyst_utils/src/ortho_camera.rs

```diff
@@ -159,7 +159,7 @@ pub enum CameraNormalizeMode {
 impl CameraNormalizeMode {
     /// Get the camera matrix offsets according to the specified options.
     fn camera_offsets(
-        &self,
+        self,
```

Comment:
why is this not borrowing anymore?

Reply:
I thought I commented that clippy says this is faster.

```
warning: this argument is passed by reference, but would be more efficient if passed by value
   --> amethyst_utils/src/ortho_camera.rs:162:9
    |
162 |         &self,
    |         ^^^^^ help: consider passing by value instead: `self`
    |
note: lint level defined here
   --> amethyst_utils/src/lib.rs:9:9
    |
9   | #![warn(clippy::all)]
    |         ^^^^^^^^^^^
    = note: #[warn(clippy::trivially_copy_pass_by_ref)] implied by #[warn(clippy::all)]
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#trivially_copy_pass_by_ref```

Reply:
Oh, it's a `Copy` type, so it doesn't actually matter much.